### PR TITLE
add back toc names for inspector

### PR DESCRIPTION
**Pull request by patricklx**
this was removed in v4

it was there until v3. 
https://github.com/ember-template-imports/ember-template-imports/blob/v3.4.2/src/babel-plugin.js#L42

with this change: 
image

### CODE REVIEW

File: src/babel-plugin.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const { basename, extname } = require('path');
+
+module.exports = function hotReplaceAst({ types: t }) {
```
Comment (lifeart) [inline src/babel-plugin.js:3]: function name seems not related to it's content

File: src/babel-plugin.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        if (
+          calleePath.referencesImport(
+            '@ember/component/template-only',
+            'default',
+          )
+        ) {
+          let params = path.node.arguments;
+          let assignment = path.parentPath.parentPath.node;
+          let rootName = basename(state.filename).slice(
```
Comment (lifeart) [inline src/babel-plugin.js:22]: i would remove endings like: `template`, `component`, `index`
Cases:
```
my-table/template.hbs
my-table/component.hbs
my-table/index.hbs
```

File: src/babel-plugin.js (hunk 1 above)
Comment (patricklx) [reply src/babel-plugin.js:3]: Oops, copied from other my other code

**Comment by lifeart** [comment]
@patricklx I'm thinking about ways to integrate it to release pipeline to not miss it in future.
We may add https://www.npmjs.com/package/babel-plugin-tester to test this babel plugin as part of test command
Here is sample: https://github.com/babel-utils/babel-plugin-tester/tree/master/test/examples (tldr - we need to create folder with 3 files, input, output, babel config - and plugin will be executed over input and compared with output)

**Review by lifeart — APPROVED** [review]

**Review by lifeart — APPROVED** [review]

**Comment by patricklx** [comment]
@lifeart what about this?

**Review by lifeart — APPROVED** [review]

**Comment by lifeart** [comment]
@patricklx looks good for me!

**Comment by patricklx** [comment]
Who can merge? @NullVoxPopuli ?

File: tests-babel/transform.test.js
Comment (NullVoxPopuli) [inline tests-babel/transform.test.js:1]: Can you add a test for classic app and addon paths?

**Review by NullVoxPopuli — CHANGES_REQUESTED** [review]
Looking good, for my confidence, however, i'd like a couple more tests for our various scenarios
- classic app
- pods app 
- classic addon
- app name is doubled up in the path instead of using the 'app' folder (broccoli thing)

**Comment by patricklx** [comment]
Mostly it doesn't matter what path is in there.
Here I'm also trying to fix the full path beeing added to the transform. That only happens under embroider.

For classic I'm depending in the project.root, parent.root to slice that off the filename.

**Review by NullVoxPopuli — APPROVED** [review]

**Comment by NullVoxPopuli** [comment]
Can you push again real quick? Ci was disabled
